.seed-config.yaml

```diff
@@ -1,21 +1,51 @@
 project:
-  name: "Dailymile"
+  name: "GoMiler"
   state: "unstable"
   image: ""
   website: ""
-  oneLiner: "Daily milestone generation for Gitlab"
+  oneLiner: "Easy milestone creation for your favorite project tracking website"
   description: |
     Generate one milestone per day\week\month within your Gitlab project.
     Use for your Gitlab based ToDo list or sprints.
-  version: "v0.0.0"
+  version: "v0.1.0"
   mainLicense: "apache2"
   docsLicense: "CCBySa"
 
 vision:
   type: "community"
   overview: |
     Simple daily milestone generation for Gitlab.
+readme:
+  usageExample: |
+  
+    git clone https://github.com/okkur/gomiler.git
```

Comment:
The usageExample seems to be quite complicated.

Why not just use:
```
go get -u -v https://github.com/okkur/gomiler/...
gomiler -namespace=yournamespace  -project=yourproject -token=123456789 -url=devhub.example.com
```

Sidenote: This assumes a correct go install, but that might not be necessary in the future and for now we can assume that. When we release binaries we can add a binary install example too.